services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall, Movie, MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+
+    movie = Movie.objects.get(id=movie_id)
```

Comment:
Using `Movie.objects.get(id=movie_id)` will raise a `DoesNotExist` exception if no movie is found with the given ID. Consider using `Movie.objects.filter(id=movie_id).first()` to handle cases where the movie might not exist.